exercises/anagram/words.json

```diff
@@ -16,7 +16,7 @@
     "does not detect a word as its own anagram":
     ["banana"],
     "does not detect an anagram if the original word is repeated":
-    ["go Go GO"],
```

Comment:
this shouldn't be in it's own commit, but rather rebased to be fixed in the original commit. the commits aren't ment for you, they are meant for everyone else, there should be no "fixing commits", but rather if your commit was wrong, alter the commit so that the development history would be clear